python/lib/bidsreader.py

```diff
@@ -93,7 +91,7 @@ def load_bids_data(self):
         force_arr     = [re.compile("_annotations\.(tsv|json)$")]
 
         # BIDSLayoutIndexer is required for PyBIDS >= 0.12.1
-        bids_pack_version = list(map(int, bids.__version__.split('.')))
+        # bids_pack_version = list(map(int, bids.__version__.split('.')))
```

Comment:
@laemtl does that change seem reasonable to you? Looks like the bids_pack_version variable is not used later in the script so I guess this is why it is commented here.

Reply:
@cmadjar I think something happen durimg a rebase since the variable is already defined L24. The commented line can be deleted. Thanks!